conf/nginx.conf

```diff
@@ -1,10 +1,16 @@
 location / {
 
+  # Force usage of https
+  if ($scheme = http) {
+    rewrite ^ https://$server_name$request_uri? permanent;
+  }
+
```

Comment:
This has been removed because it is part of the core now.

Reply:
Thanks for your remark. But why doesn't it work on my test rig, then ?
Outline app is very tricky with https usage... 

And FYI the minio app is still not working as expected, but I clearly lack of nginx knowledge to manage it by myself : 
When visiting minio domain from a browser, it should redirect to web console
When making a request to minio domain from minio client or any S3 client, it should redirect to the server (this works)
You can refer to my issue here : https://github.com/YunoHost-Apps/minio_ynh/issues/2

I would really like this to work because it would make the tests pass, both for Minio App and Outline App.
I'm kinda stuck right now, even though I continue to improve Outline app (and to use it on a daily basis on my own server)

Reply:
> When visiting minio domain from a browser, it should redirect to web console

We maybe lacking some config in `.env` file :`MINIO_BROWSER_REDIRECT_URL` cf this [post](https://geek-cookbook.funkypenguin.co.nz/recipes/minio/)

Also

> The MinIO Console is embedded as part of the MinIO Server binary starting with [RELEASE.2021-07-08T01-15-01Z](https://github.com/minio/minio/releases/tag/RELEASE.2021-07-08T01-15-01Z).

We may not need to install the console part as it is part of the MinIO server binary. cf [doc](https://docs.min.io/minio/baremetal/console/minio-console.html)

Reply:
Hi again @ericgaspar, I have been digging a little bit on the subject.
1. Apparently it's definitely impossible to have minio.url.com redirecting automatically to console or API : 
It looks like **2 different URLs are needed**
2. After checking, I got errors from minio server when trying to launch the API at minio.url.com/api and the console at minio.url.com/console.
It looks like, **2 different root URL are needed**
3. Here things start to get quite complicated for installation. Indeed, I couldn't find a way to configure two different root domains using the current install script template. The nginx helper seems to manage only one root domain.
4. I can maybe force the console domain to be a subdomain of the api domain ? Like for example, minio.url.com and console.minio.url.com ? But when I tried to add a server {} sintax into my minio.url.com domain I got errors from Nginx !
5. And should I manage to do so, I would still have troubles using yunorunner since I would need THREE root domains for my Outline app 😨 😱

I would be very thankful if you had any piece of advise here !